components/View/connector.js

```diff
@@ -6,15 +6,16 @@
  */
 
 import { connect } from 'react-redux';
-import updateHistoryState from '@shopgate/pwa-common/actions/history/updateHistoryState';
+import setViewTitle from './action-creators/setViewTitle';
+import setViewTop from './action-creators/setViewTop';
 
 /**
  * Maps the contents of the state to the component props.
  * @param {Object} state The current application state.
  * @return {Object} The extended component props.
  */
 const mapStateToProps = state => ({
-  viewTop: state.history.state.viewTop,
+  viewTop: state.ui.general.isTop,
```

Comment:
Please consider to introduce a selector for this. At least something like `getGeneralUI()`.

Reply:
Done